.setup/ansible/roles/submitty_term_creation/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Run the create_term script to create a term.
+  ansible.builtin.command:
+    cmd: ./create_term.sh --amend "{{ submitty_term_creation_key }}"
```

Comment:
Is there any plan / discussion of converting bash scripts like this to ansible? Might be beneficial to do, some of them could probably use the built in error handling and state manage ansible provides more than others

Reply:
I think this is something we should talk about. Currently, we have bash and python scripts( and a partial API too).